daq/faucet_state_collector.py

```diff
@@ -165,9 +165,42 @@ def get_switch_raw(self, switch_name):
     def get_stack_topo(self):
         """Returns formatted topology object"""
         topo_map = {}
-        topo_obj = self.topo_state
         with self.lock:
-            for link in topo_obj.get(TOPOLOGY_GRAPH, {}).get("links", []):
+            config_obj = self.system_states.get(FAUCET_CONFIG, {}).get(DPS_CFG, {})
+            links = self.topo_state.get(TOPOLOGY_GRAPH, {}).get("links", [])
+            path_to_root = self.topo_state.get(ROOT_PATH, {})
+            for dp, dp_obj in config_obj.items():
+                for iface, iface_obj in dp_obj.get("interfaces", {}).items():
+                    dp_s = iface_obj.get("stack", {}).get("dp")
```

Comment:
wtf is dp_s?  Don't explain here, fix the variable names so it's self explanatory.  peer_dp? and then dp -> start_dp?